package.json

```diff
@@ -14,8 +14,8 @@
     "node": ">=4"
   },
   "dependencies": {
-    "babel-code-frame": "7.0.0-alpha.12",
-    "babylon": "7.0.0-beta.33",
+    "babel-code-frame": "7.0.0-beta.3",
+    "babylon": "7.0.0-beta.34",
```

Comment:
FYI babylon beta versions can have breaking changes

Reply:
Based on your approval, is this one safe?

Reply:
let me know if I need to downgrade this one..

Reply:
It is safe, we didn't make any changes to babylon 33->34 besides updating our flow version internally.